packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/.run/generate_goldens.run.xml

```diff
@@ -1,7 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Are all .run scripts are deleted by mistake ?

Reply:
Woah. That was definitely NOT intended.